packages/office-ui-fabric-react/src/components/Link/__snapshots__/Link.test.tsx.snap

```diff
@@ -136,6 +195,7 @@ exports[`Link renders Link with no href as a button 1`] = `
       &:focus {
         color: #0078d4;
       }
+  disabled={undefined}
```

Comment:
Does the actual prop get rendered like this at runtime? This seems to be busted :( Maybe just a jest thing. 

@JasonGore: aria-* tags should render at all if the value is undefined, right?

Reply:
Undefined react props like this won't be passed through to the DOM, so it's the same as not having it at all.